packages/dto/src/tables/notifications/notification.rs

```diff
@@ -0,0 +1,58 @@
+use bdk::prelude::*;
+use validator::Validate;
+
+#[derive(Validate)]
+#[api_model(base = "/v1/notifications", table = notifications, action_by_id = [dismiss, update_status_to_read])]
+pub struct Notification {
+    #[api_model(summary, primary_key)]
+    pub id: i64,
+    #[api_model(summary, auto = [insert])]
+    pub created_at: i64,
+    #[api_model(summary, auto = [insert, update])]
+    pub updated_at: i64,
+
+
+    #[api_model(summary, many_to_one = users, nullable)]
+    pub target_user_id: Option<i64>,
+    #[api_model(summary, one_to_one = users)]
+    pub from_user_id: i64,
+    
+
+    #[api_model(summary, nullable)]
+    pub title: Option<String>,
+    #[api_model(summary)]
+    pub metadata: String,
+    #[api_model(summary, nullable)]
+    pub image_url: Option<String>,
+    #[api_model(summary, nullable)]
+    pub profile_url: Option<String>,
+    #[api_model(summary, nullable)]
+    pub space_id: Option<i64>,
+
+    #[api_model(summary, version = v0.1, indexed, type = INTEGER)]
+    #[serde(default)]
+    pub notification_type: NotificationType,
```

Comment:
```suggestion
    // user_id getting notification.
    #[api_model(summary, many_to_one = users)]
    pub user_id: i64,

    #[api_model(summary, type = JSONB)]
    pub metadata: NotificationData,

    #[api_model(summary)]
    pub expiration: i64,

    #[api_model(summary, version = v0.1, indexed, type = INTEGER)]
    #[serde(default)]
    pub notification_type: NotificationType,
```


Then, you should defined `NotificationData` as below

```
#[derive(Debug, Clone, Eq, PartialEq, Default)]
#[cfg_attr(feature = "server", derive(schemars::JsonSchema, aide::OperationIo))]
pub enum NotificationData {
    #[default]
    None,

    InviteTeam { team_id: i64, group_id: i64, image_url: Option<String>, description: String },
    InviteDiscussion { discussion_id: i64, image_url: Option<String>, description: String },
    BoostingSpace { space_id: i64, image_url: Option<String>, description: String },
    ConnectNetwork { requester_id: i64, image_url: String, description: String },
}
```